placement-rules-in-sql.md

```diff
@@ -1,17 +1,24 @@
 ---
 title: Placement Rules in SQL
-summary: Learn how to schedule placement of tables and partitions.
+summary: Learn how to schedule placement of tables and partitions using SQL statements.
 ---
 
 # Placement Rules in SQL
 
 > **Warning:**
 >
-> Placement Rules in SQL is an experimental feature. The syntax might change before its GA, and there might also be bugs.
->
-> If you understand the risks, you can enable this experiment feature by executing `SET GLOBAL tidb_enable_alter_placement = 1;`.
+> Placement Rules in SQL is an experimental feature introduced in v5.3.0. The syntax might change before its GA, and there might also be bugs. If you understand the risks, you can enable this experiment feature by executing `SET GLOBAL tidb_enable_alter_placement = 1;`.
+
+Placement Rules in SQL is a feature that enables you to specify where data is store in a TiKV cluster using SQL interfaces. Using this feature, consecutive data is scheduled to a specific regions, data centers, racks, or hosts, with a specified replica count. This is useful for scenarios including optimizing a high availability strategy with lower cost, ensuring that local replicas of data are available for local stale reads, and adhering to compliance requirements.
+
+The detailed user scenarios are as follows:
 
-Placement Rules allow you to configure where data will be stored in a TiKV cluster. This is useful for scenarios including optimizing a high availability strategy, ensuring that local copies of data will be available for local stale reads, and adhering to compliance requirements.
+- Place data across regions to improve the access performance in a specific region
+- Merge multiple databases of different applications to reduce the cost on database maintenance
+- Increase replica count for important data to improve the application availability and data reliability
+- Store new data into SSDs and store old data into HHDs to lower the cost on data archiving and storage
+- Schedule the leaders of hotspot data to high-performance TiKV instances
+- Separate irrelevant data to different storages to improve availability
```

Comment:
@morgo PTAL thanks! This is provided by the PM in v5.3.0 release notes.

Reply:
LGTM. Only one we need to be careful about is "Place data across regions to improve the access performance in a specific region", since this is only true for _access_ (aka stale read) scenario. For modification there are problems because the user will also need local transaction, which is a separate feature.

If PM is happy to leave this in, then I have no problems with it.

Reply:
@ThomasYYYY PTAL, thanks!

Reply:
Currently Placement Rules only solve data locality challenges. We'd better only mention read performance instead of access performance.

Reply:
"compliance" is also a big topic. Maybe we can use "data locality requirements" instead of "compliance requirements", until we figure out all the compliance requirements from real world users.
@morgo  What do you think?

Reply:
Yeah, I think that is good. We mention compliance in known limitations below, so it is good not to claim it in the description for now.

Reply:
I have replaced "compliance requirements" with "data locality requirements". PTAL